core/collector.class.inc.php

```diff
@@ -298,8 +299,10 @@ protected function DoProcessBeforeSynchro()
 			if (preg_match($sPattern, $sDataFile, $aMatches)) {
 				$idx = $aMatches[1];
 				$sOutputFile = Utils::GetDataFilePath(get_class($this).'-'.$idx.'.csv');
+
 				Utils::Log(LOG_DEBUG, "Converting '$sDataFile' to '$sOutputFile'...");
-				ini_set('auto_detect_line_endings', true);
+				//only usefull of files from old mac system deprecated in php 8.1
+				//ini_set('auto_detect_line_endings', true);
```

Comment:
Completely remove these lines then please.

Reply:
Add unit test files with Unix / Windows / Classic MacOS line endings to ensure that this still works as excepted.

Reply:
@dflaven does this line ring any bell?

Reply:
Discussed with Anne-Cath, this setting is only used by `fgets()` and `file()` which are not used in the collectors.

Reply:
The comments and commented code still needs to be removed? No?

Reply:
Yes it's done